src/components/CardDetails.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import './CardDetails.css';
+
+const CardDetails = ({ country }) => {
+  const {
+    capital,
+    subregion,
+    latlng,
+    startOfWeek,
+  } = country;
+
+  const countryCapital = capital === undefined ? 'None' : capital;
```

Comment:
- [ ] The details page all the information is held up very well! However, I can see wide space at the bottom of your app. Kindly make sure it covered all the spaces that we are seeing. Thank you! You also can paint your body  to fix!

<img width="497" alt="Screenshot 2023-02-11 at 13 35 02" src="https://user-images.githubusercontent.com/69011963/218248825-fa052a05-ee9b-4d97-a1de-943a69960982.png">